datasetAPI/put_instance_test.go

```diff
@@ -36,12 +36,15 @@ func TestSuccessfullyPutInstance(t *testing.T) {
 			os.Exit(1)
 		}
 
+
 		Convey("When a PUT request is made to update instance meta data", func() {
 			Convey("Then the instance is updated and return a status ok (200)", func() {
 				datasetAPI.PUT("/instances/{instance_id}", instanceID).
 					WithHeader(internalToken, internalTokenID).WithBytes([]byte(validPUTFullInstanceJSON)).
 					Expect().Status(http.StatusOK)
 
+					log.Debug("This instance", log.Data{"instance_ID":instanceID})
```

Comment:
I assume this debug log line should be removed and the extra whitespace once line 46 is gone.

Reply:
@nshumoogum yeah sorry, missed that.

appreciated you taking a look but I wouldn't spend any more time on this one yet. Redid main PR (approached it the wrong way) may well end up closing this and starting again.